string ring_color
int32 prize
